turbinia/config/__init__.py

```diff
@@ -53,8 +53,8 @@
     'SINGLE_RUN',
     'MOUNT_DIR_PREFIX',
     'SHARED_FILESYSTEM',
-    # TODO(aarontp): Move this to the recipe config when it's available.
     'DEBUG_TASKS',
+    # TODO(aarontp): Move this to the recipe config when it's available.
```

Comment:
I think you can just delete this line rather than moving it.

Reply:
Done.